dags/openshift_nightlies/scripts/run_benchmark.sh

```diff
@@ -12,7 +12,7 @@ setup(){
     mkdir /home/airflow/workspace
     cd /home/airflow/workspace
     echo "Cloning ${E2E_BENCHMARKING_REPO} from branch ${E2E_BENCHMARKING_BRANCH}"
-    git clone -b ${E2E_BENCHMARKING_BRANCH} ${E2E_BENCHMARKING_REPO} --depth=1 --single-branch
+    git clone -b master https://github.com/cloud-bulldozer/e2e-benchmarking.git --depth=1 --single-branch
```

Comment:
Is this intentional?

Reply:
This is for testing, I had to switch fork but will revert this before merge.